### PR TITLE
Help

### PR DESCRIPTION
**Pull request by joshuarule**

### CODE REVIEW

File: ui/controls/cascading-list.reel/cascading-list-item.reel/cascading-list-item.html [hunk 1]
```diff
@@ -24,6 +31,16 @@
...
             }
         },
+        "helpButton": {
+            "prototype": "blue-shark/ui/button.reel",
+            "properties": {
+                "element": {"#": "helpButton"},
+                "value": "Help"
+            },
+            "bindings": {
+                "classList.has('is-visible')": {"<-": "@owner.object.path(@owner.userInterfaceDescriptor.helpMessage)"}
```
Comment (thibaultzanini) [inline ui/controls/cascading-list.reel/cascading-list-item.reel/cascading-list-item.html]: Maybe you could simplify this frb expression by checking helpMessage.value is defined.

**Comment by thibaultzanini** [comment]
This branch have conflicts

**Comment by joshuarule** [comment]
Part of what I had posted this up for is to get feedback on how we are adding help messages. Currently we don't have one centralized area to add the messages...thus it will likely be hard for localization and tedious for anyone adding help documentation

**Comment by jordanhubbard** [comment]
Where does the data go now?  Is it just smeared across the various .js files?  We already know that's not going to work.  Weren't we planning on simply using GNU gettext?  
See https://docs.weblate.org/en/latest/formats.html

**Review by pchaussalet — APPROVED** [review]
LGTM

**Pull request merged**